app/src/main/java/com/hamdan/acceleromatorminiproject/domain/Score.kt

```diff
@@ -0,0 +1,20 @@
+package com.hamdan.acceleromatorminiproject.domain
+
+import com.hamdan.acceleromatorminiproject.data.ScoreEntity
+
+data class Score(
+    val dateAcquired: Long?,
+    val elapsedTime: Long?,
+)
+
+fun Score.toHighScoreEntity() = ScoreEntity(
+    dateAcquired = dateAcquired,
+    elapsedTime = elapsedTime,
+)
+
+fun ScoreEntity.toHighScore() = Score(
+    dateAcquired = dateAcquired,
```

Comment:
shouldn't you disambiguate these with a this.dateAcquired etc? Or do you feel thats unnecessary?

Reply:
I think in this case, its clear that we are assigning the entities dateAcquired as the scores dateAcquired. If there were more layers, I think this would be more appropriate to adjust